### PR TITLE
[zh] fix and update setup/production-environment/container-runtimes.md

### PR DESCRIPTION
**Pull request by zhangguanzhang**
fix and update

### CODE REVIEW

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 1]
```diff
@@ -525,24 +525,52 @@ apt-get update
 apt-get install cri-o-1.15
 ```
 {{% /tab %}}
-{{% tab name="CentOS/RHEL 7.4+" codelang="bash" %}}
+{{% tab name="CentOS/RHEL 7.4+" %}}
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:528]: This corresponds to line 283 in the English source?
The English source is only for CentOS, pls double check and sync.

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 2]
```diff
@@ -525,24 +525,52 @@ apt-get update
...
 ```shell
-# 安装必备软件
-yum-config-manager --add-repo=https://cbs.centos.org/repos/paas7-crio-115-release/x86_64/os/
+curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable.repo https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/devel:kubic:libcontainers:stable.repo
+curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable:cri-o:$VERSION.repo https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:$VERSION/$OS/devel:kubic:libcontainers:stable:cri-o:$VERSION.repo
+yum install cri-o
 ```
+-->
+
+要在以下操作系统上安装，请讲环境变量 `$OS` 设置为下表中的相应字段：
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:554]: s/讲/将/

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 3]
```diff
@@ -525,24 +525,52 @@ apt-get update
...
+
+| Operating system | $OS               |
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:556]: s/Operating system/操作系统/

File: content/zh/docs/setup/production-environment/container-runtimes.md (hunk 1 above)
Comment (zhangguanzhang) [reply content/zh/docs/setup/production-environment/container-runtimes.md:528]: 原本的table栏目比英文的要更细致分类，如果要和英文一致的话那我都修改下

Comment (zhangguanzhang) [reply content/zh/docs/setup/production-environment/container-runtimes.md]: got it

**Comment by tengqm** [comment]
Thank you!
/approve

**Review by markthink — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**